ui/job-view/pushes/Push.jsx

```diff
@@ -305,15 +305,13 @@ class Push extends React.Component {
 
     try {
       const decisionTaskId = await getGeckoDecisionTaskId(push.id, repoName);
+      const { id } = push;
+
       const jobList = await RunnableJobModel.getList(repoName, {
         decision_task_id: decisionTaskId,
+        push_id: id,
```

Comment:
Since the push ID is only used once/twice, and `id` is not specific (hard to know what it's an ID of, without context), I think it may be best to use `push.id` here and not define `id`.